rebasehelper/application.py

```diff
@@ -567,14 +567,14 @@ def build_packages(self):
                         raise RebaseHelperError('Build log %s does not exist', build_log_path)
                     except BuildLogAnalyzerMakeError:
                         raise RebaseHelperError('%s during build. Check log %s', msg, build_log_path,
-                                                logfile=build_log_path)
+                                                logfiles=build_log_path)
```

Comment:
I can see that it's not being used right now by neither `text` nor `json` output tool, but wouldn't it be better to pass it as a list, so you don't have to check type etc. After all, the argument is now called `logfiles` (plural).

Reply:
The `logfiles` attribute was meant mainly for better accessibility and will be used in HTML output tool once it makes it to the master branch.

Yes you are right, list is better. Thanks.